src/lib/orion_websocket/wsNotify.cpp

```diff
@@ -0,0 +1,51 @@
+#include "wsNotify.h"
+#include "constants.h"
+#include "parser.h"
+
+#include <libwebsockets.h>
+#include <cstring>
+
+
+std::map<std::string, lws*>client;
+
+void addSender(const std::string &subId, lws *ws)
+{
+  client[subId] = ws;
+}
+
+void removeSenders(const std::vector<std::string>& subIds)
+{
+  for (unsigned i = 0; i < subIds.size(); ++i)
+  {
+    client.erase(subIds[i]);
+  }
+}
+
+int sendNotifyContextRequestWs
```

Comment:
This function should perhaps return a bool, and it would be good to have the different conditions to return true/false (right now it returns 0/1) described in a function header